### ISSUE TITLE
NTR: Autosomal co-dominant inheritance

### ISSUE THREAD

**Issue by bpow** [obophenotype/human-phenotype-ontology#7950]
**Preferred term label:** Autosomal co-dominant inheritance

**Synonyms**
- Autosomal codominant inheritance
- Codominant autosomal inheritance
- Co-dominant autosomal inheritance

**Related ontology term** [GENO:0000143](http://purl.obolibrary.org/obo/GENO_0000143) (co-dominant autosomal inheritance)

**Definition (free text, please give PubMed ID)**
A type of autosomal dominant inheritance in which heterozygosity may result in simultaneous expression of distinct traits associated with each allele.

**Parent term (use hpo.jax.org/app)** HP:0000006

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Arguably could be applied to blood group systems like HP:0032224 (ABO) or MN blood groups. These are not diseases, *per se*, but are phenotypes.

**Your nano-attribution (ORCID)** 0000-0002-2842-8645

**Note:** could also consider a similar term "X-linked co-dominant inheritance" under HP:0001423 along the lines of [GENO:0000939](http://purl.obolibrary.org/obo/GENO_0000939) for completeness, but I am not aware of any human diseases/phenotypes that have been described with X-linked co-dominant inheritance.

**Comment by pnrobinson** [comment]
@angharadroberts thoughts?
Arguably it is enough to use "dominant" here, since "A" is dominant and "B" is dominant.

**Comment by pnrobinson** [comment]
@angharadroberts @bpow 
Sorry for the long delay in coming back to this.
This has been the community project with MOIs https://pubmed.ncbi.nlm.nih.gov/37066232/
I am not aware of cases of human disease that can be described as codominant similar to ABO. Therefore I think it would be potentially confusing to add this term to the HPO.
I will close this for now, but would be glad to have a conversation via zoom if desired.
